pyproject.toml

```diff
@@ -50,7 +50,24 @@ target-version = ["py38", "py39", "py310", "py311"]
 
 [tool.ruff]
 target-version = "py38"
-select = ["F", "E", "W", "C90", "I", "UP", "YTT", "B", "A", "C4", "T10", "RUF"]
+select = [
+  "F",
+  "E",
+  "W",
+  "C90",
+  "I",
+  "N",
+  "UP",
+  "YTT",
+  "B",
+  "A",
+  "C4",
+  "T10",
+  "RUF",
+]
+
+[tool.ruff.per-file-ignores]
+"viscm/gui.py" = ["N8"]
```

Comment:
This file is going to be a can of worms to apply pep8 naming conventions to. I think as this file is the largest by far in the repo, it will likely be slowly refactored into different modules over time as the need arises, and maybe as each small module gets extracted can have pep8 naming conventions applied at that time.